src/zenml/artifact_stores/local_artifact_store.py

```diff
@@ -149,6 +149,15 @@ def name(self) -> str:
         """
         return "local"
 
+    @property
+    def logo_url(self) -> str:
+        """A url to represent the flavor in the dashboard.
+
+        Returns:
+            The flavor logo.
+        """
+        return "https://public-flavor-logos.s3.eu-central-1.amazonaws.com/artifact_Store/local.svg"
```

Comment:
artifact_Store with capital `S`?

Reply:
should still be `s` instead of `S` right?

Reply:
ah i fixed it in the s3 bucket but not here, done now